duviz.py

```diff
@@ -246,6 +246,205 @@ def bar(self, label: str, width: int, fill='-', left='[', right=']', small='|',
         return b
 
 
+class ColumnsRenderer(TreeRenderer):
+    """
+    Render a SizeTree with vertical columns.
+    """
+
+    def __init__(
+        self,
+        height: int,
+        max_depth: int = 5,
+        size_formatter: SizeFormatter = SIZE_FORMATTER_COUNT,
+        color_mode: bool = False,
+    ):
+        super().__init__(max_depth, size_formatter)
+        self.height = height
+        self.color_mode = color_mode
```

Comment:
If I understand correctly the non color-mode is also using color escape codes ("monochrome rendering"). 
Isn't it possible to have a real non-color pure-ASCII renderer, e.g. to allow usage in context where there is no color support (e.g. a text/markdown file, ...)?

Reply:
Done, will upload new revision soon (hopefully).